src/content/api/notifications.mdoc

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
@Ray2126 I had expected us to write a guide for developers (and BNZ). Is this coming too?

API reference isn't enough. Developers need to know how to subscribe to webhooks, how to authenticate requests, best practices etc

Reply:
Good examples are Stripe and Github

Reply:
Yep sure, I had thought BNZ were only after the fields on the Notification for now. I can add a guide as well

Reply:
Added a guide now :)